recipe/meta.yaml

```diff
@@ -11,29 +11,39 @@ source:
 
 build:
   number: 0
-  noarch: python
-  script: {{ PYTHON }} -m pip install . --no-deps -vv
+  script: {{ PYTHON }} -m pip install . --no-deps --no-build-isolation -vv
+  skip: True  # [py<38]
 
 requirements:
   host:
-    - python >=3.7
+    - python
     - pip
+    - setuptools
+    - wheel
   run:
-    - python >=3.7
+    - python
     - cloudpickle
     - psutil
```

Comment:
```suggestion
```
psutil is a test dependency, not a run dependency
https://github.com/joblib/loky/blob/3.4.0/setup.py#L89

Reply:
Thanks again!